src/streamlink/plugins/artetv.py

```diff
@@ -9,24 +9,29 @@
 from streamlink.stream import HLSStream
 
 log = logging.getLogger(__name__)
-JSON_VOD_URL = "https://api.arte.tv/api/player/v1/config/{0}/{1}?platform=ARTE_NEXT"
-JSON_LIVE_URL = "https://api.arte.tv/api/player/v1/livestream/{0}"
+JSON_VOD_URL = "https://api.arte.tv/api/player/v2/config/{0}/{1}"
+JSON_LIVE_URL = "https://api.arte.tv/api/player/v2/config/{0}/LIVE"
+V2_API_BEARER = "MzYyZDYyYmM1Y2Q3ZWRlZWFjMmIyZjZjNTRiMGY4MzY4NzBhOWQ5YjE4MGQ1NGFiODJmOTFlZDQwN2FkOTZjMQ"
 
 _video_schema = validate.Schema({
-    "videoJsonPlayer": {
-        "VSR": validate.any(
-            [],
-            {
-                validate.text: {
-                    "height": int,
-                    "mediaType": validate.text,
-                    "url": validate.text,
-                    "versionProg": int,
-                    "versionLibelle": validate.text
-                },
-            },
-        )
-    }
+    "data": {
+        "attributes": {
+            "streams": validate.any(
+                [],
+                [{
+                    "url": validate.url(),
+                    "slot": int,
+                    "protocol": validate.text,
+                    "versions": validate.all(
+                        [{
+                            "label": validate.text
+                        }],
+                        validate.transform(lambda x: x[0])
+                    )
+                }]
+            )
+        }
+    },
```

Comment:
Please move the validation schema as well. Preferably directly to the `schema` argument of the `session.http.get` call, so that the whole schema does only get built when actually using the Arte plugin, as Streamlink currently loads all plugins at once.

You can add `validate.parse_json()` as the first argument of `validate.Schema()` there to avoid the `session.http.json(res, schema)` call, which is much cleaner. `validate.text` should also be replaced with a simple `str`, as it's superfluous since 2.0.0. `validate.transform(lambda x: x[0])` can be replaced with a simple `validate.get(0)`.